consumer/init_consumer.py

```diff
@@ -1,18 +1,21 @@
-import sys, os
+import os
 #import boto3
 
 from consumer import Consumer
 from kafka_message.ttypes import KafkaMessage
 from thrift.TSerialization import deserialize
 from cassandra.cluster import Cluster
 
+
+KAFKA_HOST = os.environ['KAFKA_HOST']
```

Comment:
I suggest you catch `KeyError` here and print meaningful error message to console to be able to understand why service doesn't start